docs/docs/dev/sidebar.json

```diff
@@ -87,9 +87,9 @@
       "security/api-access-control",
       "security/api-server-auth",
       "security/dp-auth",
-      "security/zone-ingress-auth",
-      "security/zoneegress-auth",
-      "security/dp-membership"
+      "security/zoneproxy-auth",
+      "security/dp-membership",
+      "security/zone-ingress-auth"
```

Comment:
Do you want to get rid of the zone-ingress-auth page completely? You can setup redirects from zone-ingress-auth and zoneegress-auth to zoneproxy-auth and then just mention that zone-ingress-auth old way is deprecated (or inline the old doc in the new doc)

Reply:
I removed the page and setup a redirect.